### DISCUSSION TITLE
The clear clipboard key

### DISCUSSION THREAD

**Discussion by ursi** [florisboard/florisboard#1724] [Feedback]
The clear clipboard key is right next to the paste key, and I think this is the worst possible place for it. Every time I paste something I have to be super careful, lest I fat finger it and delete the thing I'm trying to paste. I would like to request the ability to turn it off, or that it just be moved. I think putting it right next to copy/cut would be the ideal placement, as you're trying to destroy what's in your clipboard already.

**Comment by stpnwf** [comment]
I completely I agree. Specially since this _can_ be done through the clipboard menu already. How important is it to clear the clipboard anyway?

This is my setup. Personally I find the Clipboard Tools **way** more relevant than the Quick Actions. So I would love the option to **remove both the Clear Clipboard button and the Clipboard button** (since clipboard button is on the quick actions already - way better location imo), and have the extra real estate. Would be much better to use the left and right cursors as they wouldn't be so close together. 🙏🏼️


![Screenshot_20220403-195143_FlorisBoard-cropped](https://user-images.githubusercontent.com/81482539/161464348-a1accf4a-c455-4199-a010-73e2f282ce43.jpg)
